README.md

```diff
@@ -69,6 +69,7 @@ jobs:
           excludeTitle: 'true' # optional: this excludes the title of a pull request
           checkAllCommitMessages: 'true' # optional: this checks all commits associated with a pull request
           accessToken: ${{ secrets.GITHUB_TOKEN }} # github access token is only required if checkAllCommitMessages is true
+          excludUsers: '["user-list-to-exclude"]'
```

Comment:
let's fix the typo before merging. I believe it should be `excludeUsers` (it's missing a `e`).

You'll need to fix all references for it in code 😅